### ISSUE TITLE
Regression in latest version

### ISSUE THREAD

**Issue by MatthiasJ1** [bernhard-42/vscode-ocp-cad-viewer#81]
Measure tools does not work if 1d objects are present

```py
s = Circle(1)
show_all(measure_tools=True)
```
Ok

```py
s = Circle(1)
l = Line((0,0), (1,0))
show_all(measure_tools=True)
```
The view will be blank and the Tree tab will be glitched.

<img width="617" alt="Screenshot 2024-05-08 at 9 11 34 PM" src="https://github.com/bernhard-42/vscode-ocp-cad-viewer/assets/149267773/bea6d6af-f116-4944-ab6e-3241f1fa219b">

Notice how the Tree tab is selected but the Material tab is displayed.

**Comment by mngrm3a** [comment]
Idk if it's relevant, but this also blanks the viewer when measure tools are enabled:

```python
class APart(BasePartObject):
    def __init__(
        self,
        size: float,
        rotation: RotationLike = (0, 0, 0),
        align: Align | tuple[Align, Align, Align] = Align.CENTER,
        mode: Mode = Mode.ADD,
    ):
        part = Box(size, size, size)

        super().__init__(part, rotation, align, mode)

        RigidJoint(
            label="j1",
            to_part=self,
            joint_location=Location((0, -size / 2, 0), (90, 0, 0)),
        )


part1 = APart(5)
part2 = APart(5)
part1.joints["j1"].connect_to(part2.joints["j1"])
show_all()
```

**Comment by bernhard-42** [comment]
@mngrm3a This example works for me with and without measure tools. Could it be that you had some other variable with a line left when it showed the blank view?

**Comment by bernhard-42** [comment]
fixed in 2.4.0
